publish/publish.proj

```diff
@@ -65,6 +65,14 @@
                    Condition="'@(_MissingBlobNames)' == ''" />
     <Message Importance="High" Condition="'@(_MissingBlobNames)' != ''" Text="Skipping finalization, missing these blobs: '@(_MissingBlobNames)'" />
   </Target>
+  
+  <Target Name="UpdateVersionsRepo"
+          DependsOnTargets="FinalizeBuildInAzure"
+          Condition="'$(IsOfficialBuild)'=='true'" >
+      <MSBuild Projects="$(MSBuildThisFileDirectory)../build.proj"  Targets="UpdatePublishedVersions"
+               Properties="ShippedNuGetPackage=@(ShippedNuGetPackage);VersionsRepoPath=$(VersionsRepoPath);GitHubAuthToken=$(GitHubAuthToken);GitHubUser=$(GitHubUser);GitHubEmail=$(GitHubEmail);VersionsRepo=$(VersionsRepo);VersionsRepoOwner=$(VersionsRepoOwner)" />
```

Comment:
Have you checked to ensure that these properties are getting passed by the builds?